Docs Updates after release 6.5

DanielMurphy22 opened the pull request:
**Description of work.**
Add superplot to 06_formatting_plots page
Mention changing layout on gettingstarted/mainwindow basic course 
Add ADS import to 02_pim_sol solution code
Update 03_fit_model_choices to give better fit for EMU data

**To test:**

Build the docs-html target and check that the new links, images and wording are all correct!

Fixes #34377
Fixes #31682

*This does not require release notes* because **minor documentation update**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

DanielMurphy22 committed:
Docs Updates after release 6.5

Add superplot to 06_formatting_plots page
Mention changing layout on gettingstarted/mainwindow basic course
Add ADS import to 02_pim_sol solution code
Update 03_fit_model_choices to give better fit for EMU data

robertapplin commented on docs/source/tutorials/mantid_basic_course/fitting_data/03_fit_model_choices.rst:
```diff
@@ -18,9 +18,9 @@ Multiple peaks + background
-As can be seen in the screenshot below, the peaks don't have to be of the same type. Run the Fit
-and inspect the results.
+Zoom in on the 3 peaks around 2.15 Angstroms. Add a Linear Background, add a Lorentzian peak
+to the large central peak, and add a Gaussian peak to both of the smaller peaks., as below.
```
I think there is a typo in this line

DanielMurphy22 replied:
I assume @robertapplin means the `.,` as the typo?

robertapplin reviewed (changes requested):
There are some great additions to the docs here, nice images too :+1: 

I'm happy to approve this, I just think I spotted one typo

Commit:
Also fix #31682

Correct typo from requested change
Change AlignDetectors to ApplyDiffCal + ConvertUnits in
python_in_mantid exercise 1
Mention in python_in_mantid solutions how to get training data
Explain how to create waterfall and tiled plots

DanielMurphy22 commented:
Really sorry, I have made some more docs changes to also tackle #31682 (see the commit message for details). So @robertapplin you will also need to check the pages I have edited there, including the solutions and exercises edited!

robertapplin reviewed (approved):
Looks good to go, all the scripts run as expected :+1:
